cves/kernel/CVE-2014-7145.yml

```diff
@@ -184,8 +195,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: I am sure availability of the system is some form of requirements so when that is tampered with, it is violated.
```

Comment:
Maybe don't use first person here, and just explain how availability would be tampered with.

Reply:
updated. thanks!